applications/rag/variables.tf

```diff
@@ -152,8 +152,8 @@ variable "frontend_k8s_backend_service_port" {
 
 variable "frontend_domain" {
   type        = string
-  description = "Domain used for SSL certificate. If it's empty, *.nip.io DNS is used."
-  default     = ""
+  description = "Domain used for SSL certificate."
+  default     = "frontend.example.com"
```

Comment:
Same comments on above of defaulting this to `""` and not using `example.com` as a default. Also, I'd recommending of having multiple levels of the same defaults because it makes changes like this more complicated in the future.

Reply:
Okay we can have that.. This particluar is needed for marketplace defaults..